src/tests/hash_functions/Makefile.manual

```diff
@@ -1,3 +1,51 @@
-PROGS_SRC = test_64_bit_hash_performance.f90 test_32_bit_hash_performance.f90
+CC = gcc
+CXX = g++
```

Comment:
Should the compilers be hardwired here, or should they be determined from environmental variables. Hardwiring this doesn't guarantee it will compile if the Library proper is compiled with ifort.

Reply:
Sure, it shouldn't be hardwired. Once the issue is fixed with Intel OneAPI, we can improve this Makefile.